cpp/include/cudf/ipc.hpp

```diff
@@ -0,0 +1,28 @@
+#include <arrow/api.h>
+#include <arrow/ipc/api.h>
+#include <arrow/gpu/cuda_api.h>
+
+class CudaMessageReader : arrow::ipc::MessageReader {
```

Comment:
We prefer `snake_case` to `CamelCase` in libcudf.

Reply:
cc @trxcllnt can you make this change?

Reply:
I believe this may have been done to match Arrow's conventions as this is only a stopgap until we can move this functionality into Arrow itself.

Reply:
In that case I don't really care that much. This code doesn't interface with any of the rest of libcudf, so it's not really an issue. I'll approve just to unblock for now.

Reply:
@jrhemstad yep I was following Arrow's conventions. It's probably worth PR'ing to Arrow soon since splitting the schema and data buffers between host and device is likely a thing others will want to do.